DirectEmployers/microsite-js
# [PD-7618] - Improve/Resuable Navbar.

## What does this PR do?
@zarifrahat has expressed having a reusable navbar to utilize. Our current component had issues and no a11y features for key navigation, that said this PR:


* Adds a new refined Navbar component with left/right arrow key navigation focusing for a11y.

## Notes to Reviewer
I have put an example branch `navbar-test` you can test with in the template repo : https://github.com/DirectEmployers/microsite-template

* Clone this repo if not already cloned
* Checkout `navbar-test` branch
* Checkout this branch/PR 
* Run `npm run symlink` in the template repo to test

The home page contains the example, you should be able to tab focus the logo and from that point use left/right arrow keys to switch focus between links in the navbar.
